### ISSUE TITLE
SessionStateCredentialStore.ClearAsync throws ArgumentNullException

### ISSUE THREAD

**Issue by msvprogs** [JoeMayo/LinqToTwitter#250]
When I try to clear session variables with ClearAsync() method of SessionStateCredentialStore, it throws the following exception:
```
System.ArgumentNullException: String reference not set to an instance of a String. (Parameter 's')
   at System.Text.Encoding.GetBytes(String s)
   at Microsoft.AspNetCore.Http.SessionExtensions.SetString(ISession session, String key, String value)
   at LinqToTwitter.SessionStateCredentialStore.set_ConsumerKey(String value)
   at LinqToTwitter.SessionStateCredentialStore.ClearAsync()
```
I'm using ASP.NET Core with .NET 5.0.
It looks like you should call Remove() method on the Session for all session variable keys instead of trying to set null values to them.

Library version: LinqToTwitter.AspNet 6.7.1

**Comment by JoeMayo** [comment]
Thanks for reporting - it reproduces exactly as described. If you would like to submit a PR for this, let me know. Otherwise, I'll fix it.

Joe

**Comment by JoeMayo** [comment]
Following up - this was fixed and deployed in the latest release v6.11.0.

**Comment by msvprogs** [comment]
Thanks, it works! 👍
